styles.css

```diff
@@ -399,3 +401,140 @@ span {
 footer {
   padding-top: 20px;
 }
+
+#navbar-grid {
+  display: none;
+}
+
+#desktop-contactform {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  #navbar-grid {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+  }
+
+  #desktop-navbar {
+    grid-column: 6 / span 6;
+    justify-content: center;
+  }
+
+  #desktop-contactform {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    background: url(assets/contactbg.PNG);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    padding-bottom: 200px;
+    margin-top: 60px;
+  }
+
+  #desktop-navbar a {
+    color: #3a4a42;
+    margin-left: 15px;
+    font-weight: bold;
+  }
+
+  #projects-card {
+    grid-template-columns: repeat(3, 1fr);
+    column-gap: 20px;
+    padding-left: 20px;
+    padding-right: 20px;
+  }
```

Comment:
- Hey, great job, however, let's add some spaces to the side as seen in the images below to improve your design

*Figma design*
![image](https://github.com/AbdulSalamAbro/portfolio-website/assets/59397159/56481fac-5062-4339-8201-bc61bd6a71be)

*your design*
![image](https://github.com/AbdulSalamAbro/portfolio-website/assets/59397159/c78443b4-4e07-47c2-b6dd-ddfb2d6a07f9)